### PR TITLE
Add README.md describing inventory fields

### PR DESCRIPTION
**Pull request by bpbond**
@stephpenn1 what does "DC" stand for? That's the ForestGEO encoding right? Also, what's the `JS_codes` field? 

Name          | Description
------------- | ----------------------------------------
Plot          | Plot name (string, Control/Fresh/Salt)
Section       | Plot section (string, East/Center/West)
Tag           | Tag number (integer)
Species_code  | Species code (genus + species; https://plants.usda.gov/)
DBH_{year}    | Diameter at 1.4 m in {year}, cm (numeric)
Date_{year}   | Date inventory performed in {year} (string, YYYY-MM-DD)
Status_{year}   | Status in {year} (string, LI=live; DS=dead standing; DC=??)
Sapflux       | If sapflux tree, noted here (string)
In_Plot       | For historical reasons, some trees are out of plot (string, TRUE/FALSE)
Notes         | Notes
JS_codes      | ???

### CODE REVIEW

**Comment by stephpenn1** [comment]
It's spanish for something I don't know. I'm looking on the ForestGEO website now and ill keep you posted

**Comment by stephpenn1** [comment]
From Condit 1998, https://forestgeo.si.edu/sites/default/files/condit_1998_censusplotsmethodsbook.pdf
Still can't find the actual acronym meaning though.
Screen Shot 2021-03-23 at 3 25 18 PM

**Comment by bpbond** [comment]
Funny! Okay. It's not critical, as long as we know it means "down and dead". (Right?)

**Comment by stephpenn1** [comment]
`JS_codes` field was created by you from Jess's inventory notes. Not sure if still relevant

**Comment by bpbond** [comment]
>created by you

doh, okay, thanks

**Comment by bpbond** [comment]
@stephpenn1 thanks for your help. I've removed `JS_codes`.

File: Data/tree_inventory/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+Name          | Description
+------------- | ----------------------------------------
+Plot          | Plot name (string, Control/Fresh/Salt)
+Section       | Plot section (string, East/Center/West)
+Tag           | Tag number (integer)
+Species_code  | Species code (genus + species; https://plants.usda.gov/)
+DBH_{year}    | Diameter at 1.4 m in {year}, cm (numeric)
+Date_{year}   | Date inventory performed in {year} (string, YYYY-MM-DD)
+Status_{year} | Status in {year} (string, LI=live; DS=dead standing; DC=??)
```
Comment (stephpenn1) [inline Data/tree_inventory/README.md]: Maybe revise to: "(string, LI = living; DS = dead, stem standing; DC = dead, stem fallen)" for consistency

**Comment by bpbond** [comment]
I wonder if the "C" stands for [caído](https://www.spanishdict.com/translate/ca%C3%ADdo)

File: Data/tree_inventory/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+Sapflux       | If sapflux tree, noted here (string)
```
Comment (stephpenn1) [inline Data/tree_inventory/README.md:15]: I think it's best if we change this to `Sapflow_ID` across the board. In `ports.csv` it's labeled as `Tree_ID` which is confusing. (I'll create a separate issue for this)

**Review by stephpenn1 — CHANGES_REQUESTED** [review]

**Comment by stephpenn1** [comment]
> I wonder if the "C" stands for [caído](https://www.spanishdict.com/translate/ca%C3%ADdo)

That would make sense  ¯\_(ツ)_/¯

File: Data/tree_inventory/README.md (hunk 2 above)
Comment (bpbond) [reply Data/tree_inventory/README.md:15]: FWIW I don't see why `inventory.csv` carries this information at all. Let the sap flux datasets worry about that (i.e. let them have a single table showing the C1 maps to tree xxxx, F2 to xxxx, etc.

Comment (bpbond) [reply Data/tree_inventory/README.md:15]: But for now I will change for consistency

**Review by stephpenn1 — APPROVED** [review]

**Pull request merged**